### PR TITLE
Added guide for TS3AudioBot

### PR DESCRIPTION
**Pull request by Felix-Franz**
Added guide for TS3AudioBot

Refer to #940

### CODE REVIEW

File: source/guide_ts3audiobot.rst [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+ [isabell@stardust ts3audiobot]$ unzip TS3AudioBot_dotnet_core_3.1.zip
+ [isabell@stardust ts3audiobot]$ rm TS3AudioBot_dotnet_core_3.1.zip
+
+Now we will configure the bot.
+To set your TeamSpeak user as bot admin, you unique id is required.
+Go to your TeamSpeak client and open the `Identities` window in the `Tools` menu.
+Copy your `unique id`.
+
+To configure the bot, start it an interactively insert your configuration data (unique TeamSpeak user id, server name, server password, ...).
+After the configuration is done the bot will automatically connect to the TeamSpeak3 server.
```
Comment (luto) [inline source/guide_ts3audiobot.rst:62]:
```suggestion
To configure the bot, start it interactively and input your configuration data (unique TeamSpeak user id, server name, server password, ...).
After the configuration is done, the bot will automatically connect to the TeamSpeak3 server.
```

File: source/guide_ts3audiobot.rst [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+ .. code-block:: toml
+
+  port = 58913
+
+ to
+
+ .. code-block:: toml
+
+  port = <another port>
```
Comment (luto) [inline source/guide_ts3audiobot.rst:84]: Each guide should assume that the target uberspace is empty, so the port will be free and we don't need to handle this case here.
```suggestion
```

File: source/guide_ts3audiobot.rst [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+Installation
+============
+
+Download release file:
+
+::
+
+ [isabell@stardust ~]$ mkdir ts3audiobot
+ [isabell@stardust ~]$ cd ts3audiobot
+ [isabell@stardust ts3audiobot]$ wget https://github.com/Splamy/TS3AudioBot/releases/latest/download/TS3AudioBot_dotnet_core_3.1.zip
```
Comment (luto) [inline source/guide_ts3audiobot.rst:47]:
```suggestion
 [isabell@stardust ts3audiobot]$ wget https://github.com/Splamy/TS3AudioBot/releases/latest/download/TS3AudioBot_dotnet_core_3.1.zip
(...)
HTTP request sent, awaiting response... 200 OK
Length: 12161538 (12M) [application/octet-stream]
Saving to: ‘TS3AudioBot_dotnet_core_3.1.zip’

100%[=====================================>] 12,161,538  49.3MB/s   in 0.2s   

2021-03-14 16:09:26 (49.3 MB/s) - ‘TS3AudioBot_dotnet_core_3.1.zip’ saved [12161538/12161538]
```
Let's add some sample output so users know what to expect :)

File: source/guide_ts3audiobot.rst [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+To set your TeamSpeak user as bot admin, you unique id is required.
+Go to your TeamSpeak client and open the `Identities` window in the `Tools` menu.
+Copy your `unique id`.
+
+To configure the bot, start it an interactively insert your configuration data (unique TeamSpeak user id, server name, server password, ...).
+After the configuration is done the bot will automatically connect to the TeamSpeak3 server.
+
+::
+
+ [isabell@stardust ts3audiobot]$ dotnet TS3AudioBot.dll
```
Comment (luto) [inline source/guide_ts3audiobot.rst:66]:
```suggestion
 :emphasize-lines: 1,10,12,17,19
 
 [isabell@stardust ts3audiobot]$ dotnet TS3AudioBot.dll
 16:10:54.4473| INFO|| [============ TS3AudioBot started =============]
 16:10:54.5198| INFO|| [ Date/Time: Sunday, 14 March 2021 16:10:54
 16:10:54.5302| INFO|| [ Version: 0.11.0/master/c7e44e4d
 16:10:54.5310| INFO|| [ Platform: CentOS Linux 7 (Core) (64bit)
 16:10:54.5315| INFO|| [ Runtime: .NET Core (3.1.13) ServerGC:True GC:SustainedLowLatency
 16:10:54.5343| INFO|| [ Opus: libopus 1.0.2 (x64)
 16:10:54.5346| INFO|| [==============================================]
 16:10:55.2626| WARN|| No permission file found.
 Do you want to set up an admin in the default permission file template? [Y/n] Y
 Please enter an admin uid
 YourAdminUID
 16:11:36.1467| INFO|| Creating new permission file ({"AdminUids":["YourAdminUID"], "OverwriteIfExists":false})
 It seems like there are no bots configured.
 Fill out this quick setup to get started.
 Please enter the ip, domain or nickname (with port; default: 9987) where to connect to:
 1.3.3.7:9987
 Please enter the server password (or leave empty for none):
 SecretTSserverPWifAny
 16:12:41.2601| INFO|| Bot "default" connecting to "1.3.3.7:9987"
 16:12:41.8483| INFO|| Started Webserver on port 58913
```
The interactive mode asks quite a few questions. What do you think about adding sample output like this?

File: source/guide_ts3audiobot.rst [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+ youtube-dl = { path = "" }
+
+to
+
+.. code-block:: toml
+
+ youtube-dl = { path = "./youtube-dl" }
+
+Restart the bot to active the new config:
```
Comment (luto) [inline source/guide_ts3audiobot.rst:128]:
```suggestion
Restart the bot to activate the new config:
```

File: source/guide_ts3audiobot.rst [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+
+.. code-block:: ini
+
+ [program:ts3audiobot]
+ command=dotnet TS3AudioBot.dll
+ directory=/home/isabell/ts3audiobot/
+
+.. include:: includes/supervisord.rst
+
+Playing youtube videos is currently not possible.
```
Comment (luto) [inline source/guide_ts3audiobot.rst:107]:
```suggestion
Youtube Videos
==============

Playing youtube videos is currently not possible.
```
What do you think about starting a new section here?

File: source/guide_ts3audiobot.rst [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+If you have changed the default port above, you need to do the same here.
+
+Configure ``~/html/.htaccess`` like this:
+
+.. code-block:: htaccess
+
+ RewriteEngine On
+ RewriteRule ^(.*) http://isabell.local.uberspace.de:58913/$1 [P]
+
+You should now be able to reach the bot on ``https://isabell.uber.space``.
```
Comment (luto) [inline source/guide_ts3audiobot.rst:146]:
```suggestion
Configure the web server
========================

.. note::

    TS3AudioBot is running on port 58913.

.. include:: includes/web-backend.rst
```
Proxies using `.htaccess` are deprecated on uberspace 7, please use a [web backend](https://manual.uberspace.de/web-backends/) instead.

File: source/guide_ts3audiobot.rst [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+
+
+Uninstallation
+==============
+
+Remove reverse proxy to the web interface:
+
+::
+
+ [isabell@stardust ~]$ rm ~/html/.htaccess
```
Comment (luto) [inline source/guide_ts3audiobot.rst:235]: Please adapt this for web backends as well.

**Review by luto — CHANGES_REQUESTED** [review]
Thanks for the guide! It works well :) I added a few suggestions to fix some typos and make the guide easier to follow. Let me know what you think!

Sorry about the long wait. I was quite busy with internal projects.

**Comment by Felix-Franz** [comment]
Thanks for your suggestions @luto, I have added all of them!

**Comment by luto** [comment]
Awesome, thank you! :tada:

**Pull request merged**